packages/cli/index.js

```diff
@@ -118,10 +117,9 @@ program
     const logsFolder = path.join(rootFolder, '_markbind/logs');
     const outputFolder = path.join(rootFolder, '_site');
 
-    const defaultFiles = [INDEX_MARKDOWN_FILE, INDEX_MARKBIND_FILE];
-    const presentDefaultFile = defaultFiles.find(fsUtil.fileExists);
+    const presentDefaultFile = fsUtil.fileExists(INDEX_MARKDOWN_FILE);
```

Comment:
```suggestion
    const presentDefaultFile = fsUtil.fileExists(INDEX_MARKDOWN_FILE) ? INDEX_MARKDOWN_FILE : false;
```

Sorry my bad, this needs to be assigned the `index.md` if it exists 😓.